trade_remedies_public/core/views.py

```diff
@@ -413,13 +415,56 @@ def get(self, request, *args, **kwargs):
                 )
                 case_to_roi = {each.case.id: each for each in roi_submissions}
 
+        pending_invitation_count = 0
+        pending_invitation_deficient_docs_count = 0
+        invitation_waiting_trs_approval_count = 0
+
         # Let's get the cases where the user is awaiting approval
         invitations = v2_client.invitations(
-            contact_id=request.user.contact["id"],
-            invitation_type=2,
-            fields=["submission", "case", "invitation_type", "rejected_at", "approved_at"],
+            organisation_id=self.request.user.contact["organisation"]["id"],
+            fields=[
+                "submission",
+                "case",
+                "status",
+                "invitation_type",
+                "rejected_at",
+                "accepted_at",
+                "approved_at",
+            ],
         )
 
+        # only required if the user is an organisation owner
+        if is_org_owner:
+            pending_invitation_count = sum(
+                {
+                    1
+                    for invite in invitations
+                    if (invite.invitation_type == 1 and not invite.accepted_at)
+                    or (
+                        invite.invitation_type == 2
+                        and not invite.approved_at
+                        and not invite.rejected_at
+                        and not invite.submission.archived
+                    )
+                }
+            )
+
+            pending_invitation_deficient_docs_count = sum(
+                {
+                    1
+                    for invite in invitations
+                    if invite.invitation_type == 2 and "deficient" in invite.status
+                }
+            )
+
+            invitation_waiting_trs_approval_count = sum(
+                {
+                    1
+                    for invite in invitations
+                    if (invite.accepted_at and "waiting_tra_review" in invite.status)
```

Comment:
We need to be explicit here or we will get `NoneType` error.

```suggestion
                    if (invite.invitation_type == 2 and invite.accepted_at and "waiting_tra_review" in invite.status)
```